nss-day-cohort-41/react-nutshell-thepecans
# Complete full CRUD on events

# Description
Implemented Delete and Edit buttons for the Events, completing full CRUD.

Fixes # 5 (complete)
## Type of change

# Testing Instructions
Navigate to ".../events." Test the delete and edit buttons on existing events to ensure functionality. If no events exist, create new events, and test the buttons on the submitted new event.
# Checklist:
